### ISSUE TITLE
Deprel of Emphatic Pronouns

### ISSUE THREAD

**Issue by NathanD38** [IAHLT/UD_Hebrew#37]
@amir-zeldes

The deprel between a nominal and an emphatic pronoun in the [current HTB](https://grew-match.iahlt.org/?corpus=htb-new&custom=617a98dc6cee7) is `nmod`. 
This is the standard behavior of the parser as well.
To my understanding, this deprel cannot directly modify a nominal without the mediation of a `case`-assigned `ADP`. 

None of the English corpora actually assigns the feature `PronType=Emp`, even in cases where it clearly functions as such.
The English corpus, [GUM](http://match.grew.fr/?corpus=UD_English-GUM@2.8&custom=617a9d26b0919), assigns `nmod:npmod` to emphatic pronouns, but assigns the feature `PronType=Prs`.

In the guidelines, I erroneously equated emphatic pronouns to determiners such as **oto**, _the same one_, and
**kulo**, _the entire/whole one_, i.e., `det`-assigned `PRON`. I do not think that emphatic pronouns function, 
either syntactically or semantically, in the same manner as the aforementioned determiners. 

Should we revise the current HTB, and current and following batches, so that emphatic pronouns
receive deprel `nmod:npmod` and `PronType=Emp`?

**Comment by amir-zeldes** [comment]
Yes, that's exactly right - because we restrict nmod to PP-modifiers, we should use nmod:npmod for these. BTW that's not a universal guideline, languages that don't adopt an npmod-style subtype often use nmod without case - it's kind of a luxury to be able to reserve it just for PPs, but I think it's worth doing because it makes things much clearer. PronType=Emp is nice to have too, and probably not hard to add also for English (GUM largely tries to do the same thing as EWT with the morphology, so we just haven't gotten to it yet)

**Comment by NathanD38** [comment]
@amir-zeldes
 
Thank you! I'll add that to the revision list. :)
